spec/features/visitor_sees_list_of_murals_spec.rb

```diff
@@ -0,0 +1,19 @@
+require 'rails_helper'
+
+# [] Visiting the `/restaurants` path should contain a list of restaurants.
+# [] Visiting the `/restaurants/10` path should show the restaurant details for a restaurant with the ID of 10.
+# [] Visiting the root path should display a list of all restaurants.
+
+feature "visitor sees a list of murals" do
+  scenario "sees a list of murals" do
+    testuser = User.create(first_name: 'Name', last_name: 'last', email: 'happyboi@gmail', password: '12345678')
```

Comment:
consider the use of `create!` so that your tests yell at you if saving these records fails.